src/dcat-ap/dcat-dataset.ts

```diff
@@ -33,9 +37,16 @@ export const defaultCalculatedFields = [
   'provenance'
 ];
 
-export function getDcatDataset(hubDataset: any, orgBaseUrl: string, orgTitle: string, siteUrl: string) {
+export function getDcatDataset(hubDataset: any, orgBaseUrl: string, orgTitle: string, siteUrl: string, siteModel: IModel) {
+  const content = datasetToContent({ 
+    id: hubDataset.id, 
+    attributes: hubDataset
+  } as DatasetResource);
+  const { relative: relativePath } = getContentSiteUrls(content, siteModel);
+  const landingPage = siteUrl.startsWith('https://') ? siteUrl + relativePath : `https://${siteUrl}${relativePath}`;
```

Comment:
Why the scheme-checking logic?

Reply:
@andrewctate unfortunately this was the result of a bug from a customer's site. Their site item's URL was wacky and out of sync so Mark had to make this fix in dcat-us. Since we are changing dcat-ap to use slugs like dcat-us, it made sense to port the fix over to this repo in one go.